frontend/src/component/admin/groups/RemoveGroup/RemoveGroup.tsx

```diff
@@ -19,15 +21,14 @@ export const RemoveGroup: FC<IRemoveGroupProps> = ({
     setOpen,
     group,
 }) => {
-    const { refetchGroups } = useGroups();
     const { removeGroup } = useGroupApi();
     const { setToastData, setToastApiError } = useToast();
     const navigate = useNavigate();
 
     const onRemoveClick = async () => {
         try {
             await removeGroup(group.id);
-            refetchGroups();
+            mutate(formatApiPath('api/admin/groups'));
```

Comment:
This is super ugly, but it solves an issue we found related to deleting groups. This is currently failing our e2e tests and logs an uncaught error in production: 

<img width="1874" alt="Skjermbilde 2023-01-11 kl  15 59 01" src="https://user-images.githubusercontent.com/16081982/211839848-7c51493b-2d0f-4068-8683-47cdc7e4364b.png">

Digging locally I found that this happens because of useSWR trying run this function: https://github.com/vercel/swr/blob/main/core/use-swr.ts#L287

My theory here is that using useGroups() and exposing the data property from useSWR sets up a subscription. UseSWR needs to keep an internal cache of components to re-render once the data is refetched. My theory is that it does this by tracking the invocations of useSWR, and re-renders these components when the data changes. In this instance, the hook is used in a modal that is removed once the refetch is complete. Using the global mutate avoids setting up a subscription to re-render this function.

Reply:
isn't there some equivalent of useEffect unsubscribe function to clean this up?